distributed/utils.py

```diff
@@ -370,7 +370,7 @@ def run_loop(loop=self._loop):
             # Loop already running in other thread (user-launched)
             done_evt.wait(5)
             if not isinstance(start_exc[0], RuntimeError):
-                if not isinstance(start_exc[0]):  # track down infrequent error
+                if not isinstance(start_exc[0], Exception):  # track down infrequent error
```

Comment:
@mrocklin  Maybe launching pytest on travis with the `--showlocals` flag would help debug this kind of failures:

https://travis-ci.org/dask/distributed/jobs/438918060#L2577